Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -1178,12 +1178,8 @@ void WebPageProxy::handleSynchronousMessage(IPC::Connection& connection, const S
 
 bool WebPageProxy::hasSameGPUAndNetworkProcessPreferencesAs(const API::PageConfiguration& configuration) const
```

Comment:
We should check (in a follow-up) if we still need this function. Previously, those shared preferences could not be updated so we could not add to a WebProcess a WebPage with different preferences than the previous pages in this process. However, your new model supports updating those preferences so we may be able to reuse WebProcesses more aggressively, like we used to before my changes.

Reply:
Good point. We should reconsider that.